gw-rss-aggregator/src/main/java/ru/otus/controllers/AggregatorControllerImpl.java

```diff
@@ -0,0 +1,122 @@
+package ru.otus.controllers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.telegram.telegrambots.ApiContextInitializer;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import ru.otus.bot.commands.SubCommandResult;
+import ru.otus.bot.commands.CommandsHandler;
+import ru.otus.feeds.FeedReader;
+import ru.otus.feeds.FeedReaderProperties;
+import ru.otus.feeds.MultiFeedReader;
+import ru.otus.model.Message;
+import ru.otus.storage.Storage;
+import ru.otus.storage.StorageImpl;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+
+@Slf4j
+public class AggregatorControllerImpl implements AggregatorController {
+    private final static String STATUS_OK = "✅";
+    private final static String STATUS_ERR = "⚠️";
```

Comment:
Как-то нехорошо эмоджи прямо в исходном коде.
Лучше юникодовые коды написать:
```suggestion
    private final static String STATUS_OK = "\u2705";
    private final static String STATUS_ERR = "\u26A0";
```

Или можете посмотреть на библиотеку https://github.com/vdurmont/emoji-java

```java
String ok = EmojiParser.parseToUnicode(":white_check_mark:");
String err = EmojiParser.parseToUnicode(":warning:");  
```